cremona/src/Book.ts

```diff
@@ -0,0 +1,43 @@
+export class Book {
+    readonly name: string;
+    readonly uid: number;
+    readonly articleNumber: string;
+    readonly description: string;
+    readonly price: number;
+    readonly url: string[];
+    readonly image: string;
+    readonly courseCodes: string[];
+    readonly authors: string[];
+    readonly isbn: number;
+    readonly weight: number;
+    readonly year: number;
+
+    constructor(props) {
```

Comment:
Would it be possible to give props a type? Even if it stays as any/unkown or w.e it is, I'm fine with it. Just a small note. Or maybe I'm just mentally challenged and it isn't any/unknown.

Reply:
All we really know is that it is going to be an object; the keys and values differ depending on what book it is. I made it a bit more specific now.